src/decode.rs

```diff
@@ -4924,7 +4924,7 @@ pub(crate) unsafe fn rav1d_decode_frame(
                 rav1d_cdf_thread_update(
                     frame_hdr,
                     f.out_cdf.data.cdf,
```

Comment:
similarly here, can we just `&mut *f.out_cdf.data.cdf` here? or should other code be refactored so that this is in fact a mutable reference and not a raw mut pointer.